Model/Source/RetourInTheBox.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelNL\Magento\Model\Source;
+
+use Magento\Framework\Data\OptionSourceInterface;
+
+class RetourInTheBox implements OptionSourceInterface
+{
+    const NOT_ACTIVE        = 'notActive';
+    const NO_OPTIONS        = 'noOptions';
+    const EQUAL_TO_SHIPMENT = 'equalToShipment';
+
+    /**
+     * Options getter
+     *
+     * @return array
+     */
+    public function toOptionArray()
+    {
+        return [
+            ['value' => self::NOT_ACTIVE, 'label' => __('No')],
+            ['value' => self::NO_OPTIONS, 'label' => __('No options')],
```

Comment:
Bij WC is het veranderd naar "Without options" en "Options equal to shipment" omdat het anders niet duidelijk is

Reply:
Done